### DISCUSSION TITLE
Is it possible to use rocks.nvim with lua 5.4?

### DISCUSSION THREAD

**Discussion by Folaht** [lumen-oss/rocks.nvim#605] [Q&A] [answered]
It's the standard version available now on my linux distro.
When I try to install it with 

`luarocks --lua-version=5.4 --tree /home/user/.local/share/nvim/rocks --server='https://nvim-neorocks.github.io/rocks-binaries/' install rocks.nvim`

```
ERROR | 2024-12-14 14:06:24 | ...s/.local/share/nvim/rocks/share/lua/5.4/rocks/loader.lua:39 | ...local/share/nvim/rocks/share/lua/5.4/luarocks/loader.lua:26: module 'luarocks.core.path' not found:
```

**Comment by mrcjkb** [comment, marked as answer]
You can use luarocks with Lua 5.4, but you should still install rocks.nvim with `--lua-version=5.1`.
And if you want to be able to build luarocks packages that are compatible with Neovim, you will need either the Lua 5.1 or luajit headers.
